### PR TITLE
Resolve #1436: Add AWS Golang template, tests, and instructions on simple quick start

### LINKED ISSUE #1436: Go support
It would be great to implement Go support (maybe through a shim).

### PR DESCRIPTION
**Pull request by license2e**
Pull Request Template ------------------------
* Check out our Pull Request Guidelines here: https://github.com/serverless/serverless/blob/v1.0/CONTRIBUTING.md

##### Status:

Ready/In Development/Hold

##### Todos:

- [√] Write tests

Signed-off-by: Ilya Shindyapin <ilya@shindyapin.com>

### CODE REVIEW

**Comment by license2e** [comment]
@pmuens @flomotlik Is this on hold? Or did the CI get stuck?

**Comment by flomotlik** [comment]
@license2e not actually sure what happened here. I've restarted the build but so far its not coming in. I'll see why that happens

File: lib/plugins/create/create.js [hunk 1]
```diff
@@ -66,7 +67,7 @@ class Create {
       .log(`with template: "${this.options.template}"`);
 
     this.serverless.cli
-      .log('NOTE: Please update the "service" property in serverless.yml with your service name');
+      .log('NOTE: Please update the "service" property in serverless.yaml with your service name');
```
Comment (flomotlik) [inline lib/plugins/create/create.js]: Please revert this change is it should be yml from now on

**Comment by flomotlik** [comment]
Hmm not sure what happened here, could you change the yml/yaml thing above and push in here again and then we'll see if it triggers another build. Very strange

**Comment by license2e** [comment]
ok rebased against v1.0 and changed the note to point to `serverless.yml`

**Comment by license2e** [comment]
@flomotlik that did the trick!

**Comment by flomotlik** [comment]
@license2e the rebase somehow messed with the commit history, so a lot of changes are now shown in your PR. Could you check that out, don't know what exactly the problem here is (this happened a few times now with other contributors as well)

**Comment by license2e** [comment]
@flomotlik lol, yeah not sure, perhaps I should use `git merge` instead of `git rebase` next time? I will probably have to open up a new PR

**Comment by license2e** [comment]
@flomotlik ok, you can close this one, I opened a new one: https://github.com/serverless/serverless/pull/1712
